specification/devtestlabs/resource-manager/readme.go.md

```diff
@@ -35,3 +36,13 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 ``` yaml $(tag) == 'package-2015-05-preview' && $(go)
 output-folder: $(go-sdk-folder)/services/preview/devtestlabs/mgmt/2015-05-21-preview/$(namespace)
 ```
+
+
+### Tag: package-2018-09 and go
```

Comment:
This change is new from the already reviewed and approved PR. I was not sure if I had to add it or not. Let me know if it should be reverted.